lib/actions/splitOnJsonata.js

```diff
@@ -21,11 +21,13 @@ async function processAction(msg, cfg) {
     const results = [];
     split.forEach((elem) => results.push(elem));
     this.logger.info('Splitting the incoming message into %s messages', results.length);
-    results.forEach(async (result) => {
+
+    await Promise.all(results.map(async (result) => {
```

Comment:
This code still causes all messages to be emitted simulateously as opposed to sequentially.  Isn't sequential a requirement here?